src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.ts

```diff
@@ -29,7 +28,7 @@ export class VideoUnitComponent implements OnInit {
         if (this.videoUnit?.source) {
             // Validate the URL before displaying it
             if (this.videoUrlAllowList.some((r) => r.test(this.videoUnit.source!)) || !urlParser || urlParser.parse(this.videoUnit.source)) {
-                this.videoUrl = this.safeResourceUrlPipe.transform(this.videoUnit.source);
+                this.videoUrl = this.videoUnit.source;
```

Comment:
we should still use safeResourceUrlPipe here

Reply:
We do, I moved it to the template because it complained about it